Documents/idea document 2/ideadocument2.md

```diff
@@ -7,8 +7,38 @@
 # Methods
 (Thom)
 
-# Speficications & Execution Plan
+# Specifications & Execution Plan
 
 # Expected Outcomes
 
 # Evaluation & Outlook
+
+## Evaluation
+In the section Specifications & Execution Plan a list of MoSCoW Specifications was presented.
+These metrics are a useful and structured manner to evaluate wether or not the application is working as intended.
+The MoSCoW specifications also describe the importance of each feature.
+Each feature can then be checked if they are implemented and how well they are working.
+The must haves are the most important features and are the bare minimum for the application to present the findings on the neighborhoods.
+The should haves are expanding features on the basic functionality.
+They can be inspected on how well they filter and modify the presented data to the user.
+The could haves describe extra functionalities that can be implemented after the rest of the application is implemented properly.
+
```

Comment:
Yeah, thought the same thing. Was struggling quite a bit with how to put it, but used the Moscow definitions a bit to indicate how important they would also be for evaluation.